src/ui/panes/tag_browser.rs

```diff
@@ -199,10 +199,16 @@ impl TagBrowserPane {
                 let album = song.metadata.get("album").map_or("<no album>".to_string(), |v| {
                     v.join(&ctx.config.theme.format_tag_separator).to_string()
                 });
-                let song_date = song.metadata.get("date").map_or("<no date>", |v| v.last());
+                let song_date = ctx
+                    .config
+                    .artists
+                    .album_date_tags
+                    .iter()
+                    .find_map(|tag| song.metadata.get(tag).map(|v| v.last().to_string()))
```

Comment:
Here we can then use the derived `IntoStaticStr` from the enum.

```suggestion
                    .find_map(|tag| {
                        song.metadata
                            .get(Into::<&'static str>::into(tag))
                            .map(|v| v.last().to_string())
                    })

```